### DISCUSSION TITLE
tool-sync needs cooldown

### DISCUSSION THREAD

**Discussion by chshersh** [chshersh/tool-sync#158] [General]
Hey everyone,

During the last Hacktoberfest, I had to review tons of PRs, answer and create multiple issues in many repositories I have so I got a burnout from the amount of work I needed to do 😫 

I still plan to continue my work on `tool-sync` at some point and prepare the `v0.3.0` release but I don't know where will it happen. At the moment, there're a few open PRs in the repository that still require my review:

* https://github.com/chshersh/tool-sync/pull/153
* https://github.com/chshersh/tool-sync/pull/154
* https://github.com/chshersh/tool-sync/pull/155
* https://github.com/chshersh/tool-sync/pull/157

I'll provide my review when I'll be able to get back to this project. And when I do this, I'll ask whether you're still interested in pushing the PRs over the finish line. But of course, it's okay for you to not do any work further, it's OSS in the end 🙂 Appreciate your patience! 🙏🏻 

Anyway, this is just a quick status update. Really appreciate everyone who contributed to my little project so far! 🤗 

cc @MitchellBerend @hdhoang @gybrish @binyomen

**Comment by binyomen** [comment]
Get the rest you need, absolutely!
